src/sentry/rules/conditions/active_release.py

```diff
@@ -30,12 +31,31 @@ def is_in_active_release(self, event: Event) -> bool:
         if not event.group or not event.project:
             return False
 
-        last_release_version: Optional[str] = event.group.get_last_release()
-        if not last_release_version:
+        # XXX(gilbert):
+        # adapted from LatestReleaseFilter
+        # need to add caching later on
+        environment_id = None if self.rule is None else self.rule.environment_id
+        organization_id = event.group.project.organization_id
+        environments = None
+        if environment_id:
+            environments = [Environment.objects.get(id=environment_id)]
+
+        try:
+            latest_release_versions = get_latest_release(
+                [event.group.project],
+                environments,
+                organization_id,
+            )
```

Comment:
This gets the latest release associated with the project. Don't we want to get the release specifically associated with the event here?